README.md

```diff
@@ -110,6 +110,20 @@ lando start
 
 You can view the local environment at `localhost:8000/`. Updates to docs are automatically refreshed in the browser.
 
+## Linting and Code Formatting
+We use ESLint and Prettier to enforce code style. On each pull request to the repository, if any `.js`, `.jsx`, `.ts` or `.tsx` files are modified in the `/src` directory, ESLint will run against our Prettier configuration to check for code styling issues on the updated/changed files.
```

Comment:
> ESLint will run against our Prettier configuration to check for code styling issues on the updated/changed files.

I don't think this is accurate. The command run is `npx prettier` (which I think is the command we want)